cmd/storage-server/run.go

```diff
@@ -76,22 +78,37 @@ func Start(logger *log.Logger, port int, storePath string) (wait func(), shutdow
 		return nil, nil, err
 	}
 
-	store := diskv.New(diskv.Options{
-		BasePath:     storePath,
-		CacheSizeMax: 1024 * 1024, // 1MB cache size
-		AdvancedTransform: func(s string) *diskv.PathKey {
-			path := strings.Split(s, "/")
-			last := len(path) - 1
-			return &diskv.PathKey{
-				Path:     path[:last],
-				FileName: path[last],
-			}
-		},
-		InverseTransform: func(pk *diskv.PathKey) string {
-			return strings.Join(pk.Path, "/") + pk.FileName
-		},
-	})
-	storageServer := newStorageServer(logger, store)
+	var storageSinks []Store
+
+	gdriveStore := &gdriveServer{}
+	err = gdriveStore.Setup(logger)
+	if err != nil {
```

Comment:
It's kind of questionable to want to set up google drive by default, perhaps we want to make this an option specified at startup time from the command line

Reply:
Definitely not a default option.